src/main/java/camp/nextstep/edu/calculator/StringUtils.java

```diff
@@ -0,0 +1,10 @@
+package camp.nextstep.edu.calculator;
+
+public class StringUtils {
+    public static boolean isEmpty(final String value) {
+        if (value == null || value.isEmpty() || value.trim().length() == 0) {
```

Comment:
아래의 코드처럼 한 줄로 줄일 수 있어요.

```suggestion
        return value == null || value.isEmpty() || value.trim().length() == 0;
```

Reply:
리뷰해주신 내용을 반영하여 좀 더 간결한 구현이 되었습니다 👍